docs/Using Price Feeds/get-the-latest-price.html

```diff
@@ -96,57 +96,15 @@
 
 > 🚧 Important
 >
-> If you are using a Solidity version before 0.8.0 use [OpenZeppelin's SafeMath](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/math/SafeMath.sol).
+> If your contracts require Solidity versions that are `>=0.6.0 <0.8.0`, use [OpenZeppelin's SafeMath version 3.4](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/release-v3.4/contracts/math/SafeMath.sol).
 
 ```solidity
-// SPDX-License-Identifier: MIT
-pragma solidity 0.8.7;
-
-import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
-/**
- * THIS EXAMPLE USES UN-AUDITED CODE.
- * Network: Kovan
- * Base: BTC/USD
- * Base Address: 0x6135b13325bfC4B00278B4abC5e20bbce2D6580e
- * Quote: EUR/USD
- * Quote Address: 0x0c15Ab9A0DB086e062194c273CC79f41597Bbf13
- * Decimals: 8
- */
-
-contract PriceConverter {
-    function getDerivedPrice(address _base, address _quote, uint8 _decimals)
-        public
-        view
-        returns (int256)
-    {
-        require(_decimals > uint8(0) && _decimals <= uint8(18), "Invalid _decimals");
-        int256 decimals = int256(10 ** uint256(_decimals));
-        ( , int256 basePrice, , , ) = AggregatorV3Interface(_base).latestRoundData();
-        uint8 baseDecimals = AggregatorV3Interface(_base).decimals();
-        basePrice = scalePrice(basePrice, baseDecimals, _decimals);
-
-        ( , int256 quotePrice, , , ) = AggregatorV3Interface(_quote).latestRoundData();
-        uint8 quoteDecimals = AggregatorV3Interface(_quote).decimals();
-        quotePrice = scalePrice(quotePrice, quoteDecimals, _decimals);
-
-        return basePrice * decimals / quotePrice;
-    }
-
-    function scalePrice(int256 _price, uint8 _priceDecimals, uint8 _decimals)
-        internal
-        pure
-        returns (int256)
-    {
-        if (_priceDecimals < _decimals) {
-            return _price * int256(10 ** uint256(_decimals - _priceDecimals));
-        } else if (_priceDecimals > _decimals) {
-            return _price / int256(10 ** uint256(_priceDecimals - _decimals));
-        }
-        return _price;
-    }
-}
+{% include samples/PriceFeeds/PriceConverter.sol %}
 ```
-
+<div class="remix-callout">
+      <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/PriceFeeds/PriceConverter.sol" target="_blank" >Open in Remix</a>
+      <a href="/docs/conceptual-overview/#what-is-remix">What is Remix?</a>
+</div>
 ## How Do Data Feeds Get Their Data?
 
 Data Feeds are aggregated from many data sources by a decentralized set of independent Node Operators. The [Decentralized Data Model](../architecture-decentralized-model/) describes this in detail.
```

Comment:
```suggestion

<div class="remix-callout">
      <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/PriceFeeds/PriceConverter.sol" target="_blank" >Open in Remix</a>
      <a href="/docs/conceptual-overview/#what-is-remix">What is Remix?</a>
</div>

## How Do Data Feeds Get Their Data?

Data Feeds are aggregated from many data sources by a decentralized set of independent Node Operators. The [Decentralized Data Model](../architecture-decentralized-model/) describes this in detail.
```

Reply:
@dwightjl can you check this one please? Don't know why the callout button broke the markdown